docs/docs/getting-started/installation.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 1
+title: Installation
+---
+### Clone the project
+
+To get started, you'll need to clone the `noteable-io/kernels` project from Github.
+
+```sh
+git clone git@github.com:noteable-io/kernels
+cd kernels
+```
+
+### Requirements
+
+In order to build the various kernels provided at Noteable, you'll need a few system tools.
+
+- [Docker](https://docs.docker.com/engine/install/)
+- [Taskfile](https://taskfile.dev/installation/)
+
+Ensure the following commands return successfully before continuing
+
+```shell
+docker info
+task -h
+```
+
+### Naming Conventions
+
+All of the kernels generated by this project are named using the following format:
+
+```
+ghcr.io/noteable-io/kernel(-gpu)-${language-(version)}-$(identifier)
```

Comment:
According to your proposal, the `gpu` part should be prefixed https://docs.google.com/document/d/1y3G-BiB37KNOp_qHvLKquXKx2JkMXxaK_3Swl4627Jw/edit#bookmark=id.iq0eogau1ujp

Reply:
I updated the proposal doc to reflect the difference. The build pipelines and such are publishing to `kernel-gpu` already, and either syntax is fine